### DISCUSSION TITLE
Drive details doesn't update in the map anymore when hovering the graph.

### DISCUSSION THREAD

**Discussion by manikoutai** [teslamate-org/teslamate#2759] [General]
Hi,

In the new version 1.27, if you hover the mouse on any portion of the drive Graph from the drive details, the maps is no longer updated to indicate at what moment/place the event took places.

This was working fine in 1.26.

Anyone else is experiencing the problem or know a workaround?

Thanks

**Comment by syjnew** [comment]
Me too, after 1.27 update, hovering over drive details doesn't show places on the map. I wish it was corrected.

**Comment by LelandSindt** [comment]
https://github.com/adriankumpf/teslamate/issues/2770

**Comment by aiden-cat** [comment]
Same here - also, I get this error for tire pressure 
![image](https://user-images.githubusercontent.com/63931544/179429593-200c6cdc-3315-412b-895c-a7dc1738e2d2.png)

I didn't see specific instructions on updating tables - were they somewhere?
